src/software/ai/navigator/path_planner/enlsvg_path_planner.cpp

```diff
@@ -90,6 +90,13 @@ std::optional<Path> EnlsvgPathPlanner::findPath(const Point &start,
         return Path({start, end});
     }
 
+    // If the new unblocked points are equal, set the unblocked end point as the end point
+    // of the path
+    if (new_start == new_end)
+    {
+        return Path({start, convertEnlsvgPointToPoint(new_end.value())});
```

Comment:
should this be new_start?

Reply:
new_start is just for the enlsvg path planner to navigate outside the obstacle (if it starts in one). We want the starting position to be the robot's starting position (aka start)